packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -3121,6 +3122,18 @@ export const components = ({
             marginBlockEnd: 0,
           },
         },
+        overline: {
+          letterSpacing: 0.5,
+          textTransform: "none",
+
+          "html:lang(en) &": {
+            textTransform: "uppercase",
+          },
+
+          "html:lang(en-*) &": {
+            textTransform: "uppercase",
+          },
```

Comment:
Do we want this instead?
```suggestion
          "html:lang(en) &, html:lang(en-*) &": {
            textTransform: "uppercase",
          },
```

Reply:
@KevinGhadyani-Okta Strangely, this doesn't work. I had to make 2 declarations for this to work correctly. I don't know why

Reply:
I tried to combine these selectors in quite a few ways to get it working. No luck